edx_proctoring/api.py

```diff
@@ -1571,13 +1558,15 @@ def get_attempt_status_summary(user_id, course_id, content_id):
     # eligibility
     if credit_service and not exam['is_practice_exam']:
         credit_state = credit_service.get_credit_state(user_id, six.text_type(course_id), return_course_info=True)
-        if not _check_eligibility_of_enrollment_mode(credit_state):
+        user = User.objects.get(id=user_id)
+        if not user.has_perm('edx_proctoring.can_take_proctored_exam', exam):
             return None
 
     attempt = get_exam_attempt(exam['id'], user_id)
     if attempt:
         status = attempt['status']
-    elif not exam['is_practice_exam'] and has_due_date_passed(credit_state.get('course_end_date', None)):
+    elif not exam['is_practice_exam'] \
+            and credit_state and has_due_date_passed(credit_state.get('course_end_date', None)):
```

Comment:
nit: doesn't pep-8 prefer implicit line continuation with parentheses? I'm surprised this passes linting

Reply:
I don't know. I rely on the tool to scold me.